js-packages/@prestojs/viewmodel/src/fields/Field.ts

```diff
@@ -4,7 +4,7 @@ import { AsyncChoicesInterface } from './AsyncChoices';
 /**
  * @expand-properties
  */
-export interface FieldProps<T> {
+export interface FieldProps<T, SingleType = T> {
```

Comment:
Can you add a comment for what T and SingleType are -- it's not clear at first glance

(I assume T is the value in the field which in the case of multivalue fields would normally be an array of SingleType?)